build/media-suite_compile.sh

```diff
@@ -707,6 +708,16 @@ if [[ $standalone = y && $faac = y ]] && ! files_exist "${_check[@]}" &&
     do_checkIfExist
 fi
 
+_check=(bin-audio/exhale.exe)
+if [[ $standalone = y && $exhale = y ]] &&
+    do_vcs "https://gitlab.com/ecodis/exhale"; then
```

Comment:
```suggestion
    do_vcs "https://gitlab.com/ecodis/exhale.git"; then
```

it is preferable to use the link version that the clone button gives